src/lib/holocene/tooltip.svelte

```diff
@@ -1,28 +1,33 @@
 <script lang="ts">
-  import Copyable from './copyable.svelte';
+  import Copyable from '$lib/components/copyable.svelte';
 
   export let text: string = '';
   export let top = false;
   export let right = false;
   export let bottom = false;
   export let left = false;
   export let copyable = false;
+  export let hide: boolean | null = false;
 </script>
 
-<div class="wrapper relative inline-block">
+{#if hide}
```

Comment:
`<Tooltip />` in ui vs. holocene had diverged slightly, this is a a combo of both